core/src/components/datetime/datetime.tsx

```diff
@@ -81,6 +81,12 @@ export class Datetime implements ComponentInterface {
    */
   @Prop() displayFormat = 'MMM D, YYYY';
 
+  /**
+   * The timezone to use for display purposes only. By default it will display values
+   * in the user's timezone.
+   */
+  @Prop() displayTimezone?: string;
```

Comment:
What are the accepted values here? Maybe link to https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toLocaleString